src/internals/tooltip.js

```diff
@@ -246,18 +246,70 @@ extend(ChartInternal.prototype, {
 		const tWidth = $$.tooltip.property("offsetWidth");
 		const tHeight = $$.tooltip.property("offsetHeight");
 		const position = positionFunction.call(this, dataToShow, tWidth, tHeight, element);
+		const x = selectedData[0].x || undefined;
 
 		// Set tooltip
 		$$.tooltip
 			.style("top", `${position.top}px`)
 			.style("left", `${position.left}px`);
+
+		if (config.tooltip_linked) {
+			$$.api.internal.charts.forEach(c => {
```

Comment:
Most of this block is duplicated on hide also.
It should making as a separated function to handle both states (showing & hiding).

Reply:
Moving this to a separate handler function.